redis-pool-store/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "tower-sessions-redis-pool-store"
+description = "Redis session store with pool support. Not for direct use; see the `tower-sessions` crate for details."
+documentation.workspace = true
+version.workspace = true
+license.workspace = true
+edition.workspace = true
+authors.workspace = true
+repository.workspace = true
+
+[dependencies]
+async-trait = { workspace = true }
+redis_pool = "0.2.1"
+redis = "0.23.4"
```

Comment:
I'm okay to support fred and redis but this is a confusing interface. Both stores are currently called "redis" but depend on different crates for the client.

Reply:
I agree, in fact I wanted to discuss the integration before investing too much time on it.

The exposed features needs to define which client they are using (eg. `redis_fred`) without being reduntant (eg. `redis_redis`).
Maybe renaming `RedisStore` to `RedisFredStore` could also help with clarity at the cost of compatibility.

All these thoughts come from the prospective of not merging the interfaces like `sqlx-store`,
but I'm open to reconsider my position😄

Reply:
I think we'll want to resolve https://github.com/maxcountryman/tower-sessions/issues/102 to help address this.